lib/types.js

```diff
@@ -0,0 +1,64 @@
+const isNumber = val => typeof val === 'number';
+const isString = str => typeof str === 'string';
+const isBoolean = bool => typeof bool === 'boolean';
+const isArray = arr => Array.isArray(arr);
+const isObject = obj => Object.prototype.toString.call(obj) === '[object Object]';
```

Comment:
Clever use of prototype. You should be able to use String() instead to get the same result.